project-linter/rules/src/main/scala/fix/RestrictedImports.scala

```diff
@@ -0,0 +1,63 @@
+package fix
+
+import scalafix.v1._
+
+import scala.meta._
+
+object RestrictedImports {
+
+  /** If a file is within any of the packages defined below, it will not be checked for restricted imports.
+    */
+  private val ignoredPackages: Seq[String] = Seq("io.shiftleft.semanticcp")
+
+  /** The rules for restricted imports.
+    */
+  private val rules: Map[String, ImportRule] = Seq(
+    ImportRule(
+      "BetterFiles",
+      "Avoid the 'Better Files' library, use `io.joern.x2cpg.utils.FileUtil` or `java.nio.file` instead.",
```

Comment:
```suggestion
      "Avoid the 'Better Files' library, use `io.joern.x2cpg.utils.FileUtil`, `os-lib` or `java.nio.file` instead.",
```

Reply:
Also update `io.joern.x2cpg.utils.FileUtil` to `io.shiftleft.semanticcpg.utils.FileUtil`